ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -18,6 +18,12 @@ if [[ ! -s "${SHARED_DIR}/metadata.json" ]]; then
   exit
 fi
 
+echo ${SHARED_DIR}/metadata.json
+
+if [[ "${CLUSTER_TYPE}" == "azurestack" ]]; then
+  export AZURE_AUTH_LOCATION=$SHARED_DIR/osServicePrincipal.json
+fi
+
```

Comment:
Instead of adding this conditional, can we change the workflow to point to a different deprovision step?

Reply:
Isn't this required for the installation process too?

Reply:
I'm not sure what you mean. Is this what you're referring to: https://github.com/openshift/release/pull/18796/commits/0f3315a296894ef685ece22e1924dba30d2e2701#diff-65cce809dc5ddaa91711f6709acd41e5c24b735c073708e60f10aae28faf8712R65

The point I'm trying to make is that with  steps we want to avoid branching or conditionality as much as possible within the steps. I think this is fine to include initially. Let's not worry about it.